raiden/network/blockchain_service.py

```diff
@@ -49,6 +49,9 @@ def __init__(
     def block_number(self) -> int:
         return self.client.block_number()
 
+    def get_block(self, block_identifier) -> int:
```

Comment:
the `int` annotation is wrong, it will be a [dict](https://web3py.readthedocs.io/en/stable/web3.eth.html?highlight=getBlock#web3.eth.Eth.getBlock)